website/common/script/i18n.js

```diff
@@ -60,4 +61,26 @@ function t (stringName) {
   }
 }
 
+// translates passed strings to two (passed) languages and concatenates them
+// returns an array
+function translateTwice (languages, messages) {
```

Comment:
I don't know. Because we're creating a completely new function with its own logic and test requirements for a specific situation that it's probably not going to be used elsewhere